### PR TITLE
Install antd; closes #2342.

### LINKED ISSUE #2342: install antd
- replace blueprint && WP components with antd equivalents

- use adapters? ie: EE versions that wrap antd component

- create PRs for separate areas of src, ie: `application/ui/inputs`

### PR DESCRIPTION
**Pull request by nicolad**
## Problem this Pull Request solves
- replace blueprint && WP components with antd equivalents

- use adapters? ie: EE versions that wrap antd component

- create PRs for separate areas of src, ie: `application/ui/inputs`

- closes https://github.com/eventespresso/event-espresso-core/issues/2342

### CODE REVIEW

File: babel.config.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
 
-	const plugins = ['@babel/plugin-proposal-class-properties', '@babel/plugin-transform-runtime'];
+	const plugins = [
+		'@babel/plugin-proposal-class-properties',
+		'@babel/plugin-transform-runtime',
+		[
+			'import',
+			{
+				libraryName: 'antd',
+				style: 'css', // or 'css'
```
Comment (manzoorwanijk) [inline babel.config.js]: Since we have loader for less, we can set it to `style: true` which is faster along with `libraryDirectory: 'es'`

Comment (nicolad) [reply babel.config.js]: We don't have yet `less` loader, just `sass/scss` loader.

File: config/resolve-tsconfig-path-to-webpack-alias.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+		const key = item.replace('/*', '');
+		const value = resolve('./', paths[item][0].replace('/*', '').replace('*', ''));
+
+		aliases[key] = value;
+	});
+
+	return aliases;
+}
+
+module.exports = resolveTsconfigPathsToAlias;
```
Comment (nicolad) [inline config/resolve-tsconfig-path-to-webpack-alias.js]: Got inspired from here: https://gist.github.com/nerdyman/2f97b24ab826623bff9202750013f99e

File: config/webpack.config.js [hunk 3]
```diff
@@ -612,7 +601,7 @@ module.exports = function(webpackEnv) {
 					// The formatter is invoked directly in WebpackDevServerUtils during development
 					formatter: isEnvProduction ? typescriptFormatter : undefined,
 				}),
-			// ...pluginsConfigWithExternals,
+			new webpack.ContextReplacementPlugin(/moment[\/\\]locale$/, /(en)$/),
```
Comment (manzoorwanijk) [inline config/webpack.config.js]: 👍

File: js-green-licenses.json [hunk 4]
```diff
@@ -20,6 +20,7 @@
 		"@svgr/webpack",
 		"@typeform/embed",
 		"caniuse-lite",
+		"component-indexof",
```
Comment (nicolad) [inline js-green-licenses.json]: MIT license: https://github.com/component/indexof/blob/master/LICENSE

Comment (nicolad) [reply js-green-licenses.json]: image

**Comment by nicolad** [comment]
A follow-up PR will be created to adjust spacing, not in scope of current PR.

**Pull request merged**